airbyte-integrations/connectors/source-intercom/source_intercom/source.py

```diff
@@ -280,10 +286,12 @@ class Conversations(IncrementalIntercomStream):
 
     data_fields = ["conversations"]
 
-    def request_params(self, next_page_token: Mapping[str, Any] = None, **kwargs) -> MutableMapping[str, Any]:
-        params = super().request_params(next_page_token, **kwargs)
-        params.update({"order": "asc", "sort": self.cursor_field})
-        return params
```

Comment:
I guessed you removed this because default ordering is `{"order": "desc", "sort": "updated_at"}`. I think you should rather explicitly set `params.update({"order": "desc", "sort": self.cursor_field})` for safety in case of API changes and to improve the understanding of the code.